src/main/java/htsjdk/utils/NullWriter.java

```diff
@@ -0,0 +1,32 @@
+package htsjdk.utils;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.Writer;
+
+/**
+ * Implementation of a writer that does nothing
+ */
+public class NullWriter extends Writer {
```

Comment:
Should this be in the main or the test sources?

Reply:
main....its used in the production code

Reply:
I am not sure it belongs in htsjdk and not in your production code.

Reply:
Until you have more than one use case in htsjdk, it could live inside the writer class as a static private inner class.